packages/components/src/__alpha__/SingleSelect/_docs/SingleSelect.stories.tsx

```diff
@@ -12,7 +13,9 @@ export default meta
 type Story = StoryObj<typeof meta>
 
 export const Playground: Story = {
-  args: {},
+  args: {
+    items: singleMockItems,
```

Comment:
I noticed in the playground that when there is no children there appears to be a bullet rendered. 
<img width="261" alt="Screenshot 2025-07-08 at 11 10 09 am" src="https://github.com/user-attachments/assets/66bede5b-050c-427b-b23d-72477d2152b7" />

I'm assume we'll tackle what null or an empty children prop looks like later but just wanted to flag it in this story :)